packages/story-editor/src/components/floatingMenu/elements/shared/color.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { PLACEMENT } from '@googleforcreators/design-system';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Color from '../../../form/color';
+
+const EYEDROPPER_WIDTH = 38; // icon width + spacing
+const WIDTH_INCLUDING_INPUTS = 178;
+const WIDTH_EXCLUDING_INPUTS = 60;
+
+function getWidth(hasInputs, hasEyedropper) {
+  return (
+    (hasInputs ? WIDTH_INCLUDING_INPUTS : WIDTH_EXCLUDING_INPUTS) +
+    (hasEyedropper ? EYEDROPPER_WIDTH : 0)
+  );
+}
+
+function FloatingColor(props) {
+  const { hasInputs, hasEyedropper } = props;
+  const width = getWidth(hasInputs, hasEyedropper);
+  return (
+    <Color
+      width={width}
+      maxHeight={362}
+      pickerPlacement={PLACEMENT.TOP_END}
+      isMinimal
```

Comment:
The whole item is compressed in size when displayed with inputs (and optionally opacity too). Compare the menu with and without `isMinimal` below (when `hasInputs` is true):

| `isMinimal` | Screenshot |
|-|-|
| ✅  | ![Screen Shot 2022-03-02 at 15 24 21](https://user-images.githubusercontent.com/637548/156443802-19d3b746-3e3c-4d8b-b771-4cdcbb9ff017.png) |
| ❌  | ![Screen Shot 2022-03-02 at 15 19 23](https://user-images.githubusercontent.com/637548/156443797-e700bf42-b398-40d3-aa86-1b6b3b1eef7f.png) |

Alternate names are welcome. We could simply name it `isInDesignMenu` as this should always be true if in the design menu and always false if not.

Reply:
> We could simply name it `isInDesignMenu` as this should always be true if in the design menu and always false if not.

Yep, something that makes it more clear when to use it would be great, `isInDesignMenu` sgtm.